[Bug] Compilation error with Angular 13

```
Error: node_modules/@editorjs/editorjs/types/api/tooltip.d.ts:4:9 - error TS2614: Module '"codex-tooltip"' has no exported member 'TooltipContent'. Did you mean to use 'import TooltipContent from "codex-tooltip"' instead?

Error: node_modules/@editorjs/editorjs/types/api/tooltip.d.ts:4:25 - error TS2614: Module '"codex-tooltip"' has no exported member 'TooltipOptions'. Did you mean to use 'import TooltipOptions from "codex-tooltip"' instead?
```

Steps to reproduce:
1. ```import EditorJS from '@editorjs/editorjs';```
2. ```this._editor = new EditorJS({holder: this.id});```

Editor.js version: 2.22.2

Angular version: 13.0.2

Plugins you use with their versions: none

Error: node_modules/@editorjs/editorjs/types/api/tooltip.d.ts:4:9 - error TS2614: Module '"codex-tooltip"' has no exported member 'TooltipContent'. Did you mean to use 'import TooltipContent from "codex-tooltip"' 
instead?

Solution Of This 
Change in node_modules/@editorjs/editorjs/types/api/tooltip.d.ts file.
Replace this line
import {TooltipContent, TooltipOptions} from 'codex-tooltip';
to these
import { TooltipContent } from 'codex-tooltip/dist/types';
import { TooltipOptions } from 'codex-tooltip/dist/types';

Same issue
Angular: 13
Editorjs: 2.23.2

```
Error: node_modules/@editorjs/editorjs/types/api/tooltip.d.ts:4:9 - error TS2614: Module '"codex-tooltip"' has no exported member 'TooltipContent'. Did you mean to use 'import TooltipContent from "codex-tooltip"' instead?

import {TooltipContent, TooltipOptions} from 'codex-tooltip';


Error: node_modules/@editorjs/editorjs/types/api/tooltip.d.ts:4:25 - error TS2614: Module '"codex-tooltip"' has no exported member 'TooltipOptions'. Did you mean to use 'import TooltipOptions from "codex-tooltip"' instead?

import {TooltipContent, TooltipOptions} from 'codex-tooltip';
```

--- a/package.json
+++ b/package.json
@@ -97,7 +97,7 @@
   },
   "dependencies": {
     "codex-notifier": "^1.1.2",
-    "codex-tooltip": "^1.0.4",
+    "codex-tooltip": "^1.0.5",
     "nanoid": "^3.1.22"
   }
 }

